plugins/org.jkiss.dbeaver.ext.altibase.ui/plugin.xml

```diff
@@ -72,5 +72,55 @@
             <supports dataSource="altibase"/>
         </editorContribution>
     </extension>
+    
+    <extension point="org.jkiss.dbeaver.tools">
+        <tools>
+            <tool
+                  description="%tool.org.jkiss.dbeaver.ext.altibase.ui.tools.maintenance.AltibaseToolMViewRefresh.label"
+                  group="org.jkiss.dbeaver.ext.altibase.ui.tools.maintenance"
+                  icon="#refresh"
+                  id="org.jkiss.dbeaver.ext.altibase.ui.tools.maintenance.AltibaseToolMViewRefresh"
+                  label="%tool.org.jkiss.dbeaver.ext.altibase.ui.tools.maintenance.AltibaseToolMViewRefresh.label"
+                  singleton="false">
+                <task id="altibaseToolMViewRefresh"/>
+            </tool>
+            <tool
+                  description="%tool.org.jkiss.dbeaver.ext.altibase.ui.tools.maintenance.AltibaseToolTruncateTable.label"
+                  group="org.jkiss.dbeaver.ext.altibase.ui.tools.maintenance"
+                  icon="#delete"
+                  id="org.jkiss.dbeaver.ext.altibase.ui.tools.maintenance.AltibaseToolTableTruncate"
+                  label="%tool.org.jkiss.dbeaver.ext.altibase.ui.tools.maintenance.AltibaseToolTruncateTable.label"
+                  singleton="false">
+                <task id="altibaseToolTableTruncate"/>
+            </tool>
+            <tool
+                  description="%tool.org.jkiss.dbeaver.ext.altibase.ui.tools.maintenance.AltibaseViewCompile.label"
+                  group="org.jkiss.dbeaver.ext.altibase.ui.tools.maintenance"
+                  icon="icons/compile.png"
```

Comment:
You can use `platform:/plugin/org.jkiss.dbeaver.ui/icons/misc/compile.png` instead

Reply:
Thanks for your tip. 

At the first time, I used that icon but that icon didn't seem to fit the action I intended. If I can't solve the problem, I'll follow your advice :)